### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by Dove-white**
# Summary

- Install react-router
- Perform routing in the App component
- Add a "create-new" page
- Add a nav component

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+import CreateNews from "./pages/CreateNews";
 
 function App() {
   return (
     <>
-      <Overview />
+      <Nav />
+      <Routes>
+        <Route path="/" element={<Overview />} />
+        <Route path="/addNews" element={<CreateNews />} />
```
Comment (SekoViper) [inline src/App.jsx]: Kindly format your path in a way that meets the industry standard. E.g `add-news` not addNews.

File: src/component/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Nav = () => {
+    const className = "font-medium text-black hover:underline cursor-pointer";
```
Comment (SekoViper) [inline src/component/Nav.jsx]: Just write styles for each element. That is a better approach.

File: src/component/Nav.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <>
+      <nav className="p-5 xl:px-[10%] flex justify-end w-full">
+        <ul className="flex gap-4">
+          <li>
+            <Link className={className} to="/">Views</Link>
```
Comment (SekoViper) [inline src/component/Nav.jsx]: Rename the link name to News instead.

File: src/component/ui/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      action="#"
+      className="w-2/5 max-lg:w-2/3 max-sm:w-[90%] flex flex-col gap-7"
+    >
+      <InputFilled
+        label="Title"
+        type="text"
+        idName="news_title"
+        placeholder="Breaking News"
+      />
+      <InputFilled
```
Comment (SekoViper) [inline src/component/ui/Form.jsx:18]: Your content filed should not be an input field but rather a textarea element.

File: src/component/ui/InputFilled.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const InputFilled = ({label, type, idName, placeholder}) => {
+  return (
+    <>
+      <div>
+        <label
+          for={idName}
```
Comment (SekoViper) [inline src/component/ui/InputFilled.jsx]: Please rename your prop idName to id. id best explain what it is used for?

**Review by SekoViper — CHANGES_REQUESTED** [review]
# Require changes 🧧 
Kindly look at the comments to improve your project. You are almost there. Thanks.

**Comment by Dove-white** [comment]
Thanks for the review, the changes requested will be done soon.

**Review by SekoViper — APPROVED** [review]
# Approved 🟢 
Hello @Dove-white ,

Congratulations 😆 🏆 .
You have done a great job to complete this part of the project.
Happy coding as you continue your project. 

Thanks.

**Comment by Dove-white** [comment]
Welcome

**Pull request merged**